### PR TITLE
Create initial contrbuting guides

### PR DESCRIPTION
**Pull request by rbtr**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Comment by pjohnst5** [comment]
Hi @rbtr, this is awesome, thanks

How do I sign the CLA? Might be why my check is yellow here:
image

**Comment by rbtr** [comment]
https://github.com/microsoft/contributorlicenseagreement#accepting

however, you have probably already signed the CLA and that's just a flake. see the checks your previous PR [#2205](https://github.com/Azure/azure-container-networking/pull/2205/checks?check_run_id=16688901250)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Contributing guidelines
+Contributers must sign the CLA. The instructions will be provided to new contributors when their first PR is opened, as a comment.
```
Comment (pjohnst5) [inline CONTRIBUTING.md:2]: Can we say also, "If you are having problems with the CLA, comment `@microsoft-github-policy-service rerun` on your PR to trigger a re-run

**Comment by jpayne3506** [comment]
For historical purposes, commenting `@microsoft-github-policy-service rerun` will re-run the CLA check on your individual PR's.

File: CONTRIBUTING.md (hunk 1 above)
Comment (pjohnst5) [reply CONTRIBUTING.md:2]: Otherwise looks great to me

**Review by pjohnst5 — CHANGES_REQUESTED** [review]
Just one thing to add

**Review by ramiro-gamarra — APPROVED** [review]
lgtm

**Pull request merged**